tide_core.module

```diff
@@ -256,12 +256,27 @@ function tide_core_preprocess_status_messages(&$variables) {
  */
 function tide_core_form_content_moderation_entity_moderation_form_alter(&$form, FormStateInterface $form_state, $form_id) {
   if (isset($form['revision_log'])) {
-    $form['#attached']['library'] = 'tide_core/content_moderation';
+    $form['#attached']['library'][] = 'tide_core/content_moderation';
+    $form['#attached']['library'][] = 'tide_core/node_revision_log';
     $form['revision_log']['#type'] = 'textarea';
-    $form['revision_log']['#title'] = t('Publishing instructions and comments');
-    $form['revision_log']['#description'] = t('All content will be published within 24 hours. State below if your content is embargoed or time critical.');
-    $form['new_state']['#title'] = t('Change status to');
+    $form['revision_log']['#title'] = t('Change request detail');
+    $form['revision_log']['#description'] = t('Specify details of <strong>all changes</strong> made in this update. If content is embargoed or time specific, enter the scheduling details. Content will be published within <strong>24 business hours</strong>. Contact your content team for urgent publishing requests.');
+    $form['#validate'][] = '_tide_core_node_form_log_message_validate';
   }
+
+}
+
+/**
+ * Moderation form validate function.
```

Comment:
This is probably going to need a full docblock comment to pass code standards, isn't it?

Something like -- 

```
/**
 * Node form #validate callback.
 *
 * @param array $form
 *   Form that is being processed.
 * @param \Drupal\Core\Form\FormStateInterface $form_state
 *   The current state of the form.
 *
 * @return void
 */
```


Not exactly that, but something like that.